openff/interchange/components/interchange.py

```diff
@@ -414,6 +414,14 @@ def to_openmm_system(
         system : openmm.System
             The OpenMM System object.
 
+        Notes
+        -----
+        There are some sharp edges and quirks when using this method. Be aware of some documented
+        issues in the "Sharp edges" section of the user guide. If you encounter surprising
+        behavior that is not documented, please raise an issue.
+
+        https://docs.openforcefield.org/projects/interchange/en/stable/using/edges.html
+
```

Comment:
Is this on the right method? The edges described are in `from_openmm` and `to_gromacs`, but this is `to_openmm_system`

```suggestion
        Notes
        -----
        There are some sharp edges and quirks when using this method. Be aware of some documented
        issues in the :doc:`/using/edges` section of the user guide. If you encounter surprising
        behavior that is not documented, please raise an issue.

```

Reply:
Wonderful, I wasn't sure how to do that